projects/igniteui-angular/src/lib/services/overlay/utilities.ts

```diff
@@ -65,6 +65,9 @@ export interface OverlayEventArgs {
 export interface OverlayCancelableEventArgs extends OverlayEventArgs, CancelableEventArgs {
 }
 
+export interface OverlayClosingEventArgs extends OverlayEventArgs, CancelableBrowserEventArgs {
```

Comment:
This should be added to exports in `services/index.ts` so it can be included in the public API.